docs/stable/1-client.md

```diff
@@ -94,33 +96,33 @@ To communicate between Status nodes, the [RLPx Transport
 Protocol, v5](https://github.com/ethereum/devp2p/blob/master/rlpx.md) is used, which
 allows for TCP-based communication between nodes.
 
-On top of this we run the RLPx-based subprotocol [Whisper
-v6](https://eips.ethereum.org/EIPS/eip-627) for privacy-preserving messaging.
-
-There MUST be a node that is capable of discovering peers and
-implements Whisper V6 specification.
+On top of this we run RLPx-based subprotocols, the client
+MAY use [Whisper v6](https://eips.ethereum.org/EIPS/eip-627), though the client 
+SHOULD use [Waku V1](https://github.com/vacp2p/specs/blob/master/specs/waku/waku-1.md)
+for privacy-preserving messaging.
```

Comment:
Perhaps add a sentence on why Waku should be used, like "Waku should be used because it preserves your bandwidth better". Additionally, we might want to say here that Whisper SHOULD NOT be used to indicate that it is a compatibility thing.

Reply:
A point of clarification. Are you suggesting to change:

"MAY use Whisper" to "SHOULD NOT use Whisper"?